pkg/local_object_storage/blobstor/blobovnicza.go

```diff
@@ -628,7 +628,7 @@ func (b *blobovniczas) getObjectRange(blz *blobovnicza.Blobovnicza, prm GetRange
 	to := from + prm.rng.GetLength()
 	payload := obj.Payload()
 
-	if uint64(len(payload)) < to {
+	if pLen := uint64(len(payload)); to < from || pLen < from || pLen < to {
```

Comment:
`to` equals `from` (uint) + `len` (uint) so i doubt it could be less than `from`

Reply:
Try `uint64(10) + math.MaxUint64`

Reply:
oh, overflow...